apparmor/OWNERS

```diff
@@ -0,0 +1,3 @@
+reviewers:
+- ingvagabund
```

Comment:
@timothysc, as far as I understood manually opting in reviewers was out of scope for the current work. I'm happy to work on adding that ability if you think it's necessary. I'd also love if @dchen1107 replied to my robot email to signal preferences for reviewing kubernetes/kubernetes :smile:. Also several people have communicated preference to review paths not identified by examining commits so we probably need to have a broader discussion about how to get added to an OWNERS file outside of commit history